contrib/babelfishpg_tsql/src/guc.c

```diff
@@ -42,8 +42,7 @@ int   pltsql_datefirst = 7;
 int   pltsql_rowcount = 0;
 char* pltsql_language = NULL;
 int pltsql_lock_timeout = -1;
-bool pltsql_enable_linked_servers = true;
-bool pltsql_allow_windows_login = true;
+bool pltsql_allow_windows_login = false;
```

Comment:
pltsql_enable_linked_servers should not be removed.

Reply:
Done!